apps/api/drizzle/0062_yummy_paladin.sql

```diff
@@ -39,12 +39,12 @@ CREATE TABLE "seasons" (
 );
 --> statement-breakpoint
 INSERT INTO seasons (number, name, start_date, end_date)
-	VALUES (0, 'Genesis', '2025-10-13T00:00:00Z', null)
+	VALUES (0, 'Genesis', '2025-10-14 17:00:00-07', '2025-11-14 17:00:00-07')
```

Comment:
Ok so we're going with 10:00UTC? That should work, just wondering if there is a reason why.

Reply:
Ah, we ran the genesis season with and end time of 2025-11-15 00:00+00 which is 2025-11-14 17:00:00-07. Got it. Should we just use UTC time in here to avoid confusion? Also, should we insert season 1 at least as well?

Reply:
I ended up setting these to exactly what is in production to avoid confusion.  Also this won't have any affect since the data already exists.